MovieQuiz/Services/StatisticService.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  StatisticService.swift
+//  MovieQuiz
+//
+//  Created by Aleksandr Eliseev on 03.11.2022.
+//
+
+import Foundation
+
+protocol StatisticServiceProtocol {
+    var totalAccuracy: Double { get }
+    var gamesCount: Int { get }
+    var bestGame: GameRecord { get }
+    
+    func store(correct count: Int, total amount: Int)
+}
+
+final class StatisticServiceImplementation: StatisticServiceProtocol {
+    
+    private var gameRecord: GameRecord?
+    
+    private(set) var totalCorrect: Int {
+        get {
+            return userDefaults.integer(forKey: Keys.correct.rawValue)
+        } set {
```

Comment:
get/set с новой строки всегда
```suggestion
        } 
        set {
```